README.md

```diff
@@ -49,14 +40,13 @@ already comfortable with `tasty-quickcheck` and property testing in general; for
 those who are not, or those who prefer to learn from examples, we provide three
 executable examples:
 
-* `examples/square-tests`, which demonstrates basic usage of both universal and
-  conditional properties;
-* `examples/zip-tests`, which demonstrates conditional properties with crashing
-  cases; and
-* `examples/natural-tests`, which demonstrates law checking.
+* `examples/reverse-tests`, which demonstrates how to define property 
+using definition of behavior in Haskell. 
+* `examples/zip-tests`, which demonstrates how to use `fromPFun` to 
+define Plutarch property. 
```

Comment:
'how to define a property'
'using a Haskell definition for intended behaviour'
'to define a Plutarch property'